robocop/utils/utils.py

```diff
@@ -69,11 +69,11 @@ def issues_to_lsp_diagnostic(issues):
     return [{
         'range': {
             'start': {
-                'line': issue.line,
+                'line': issue.line - 1,
```

Comment:
This needs to be `'line': issue.line - 1 if issue.line > 0 else 0` (it seems there are some cases where 0 is reported and it became negative... in particular the: `Missing documentation in suite`).